libs/application/templates/financial-aid/src/fields/SummaryForm/SummaryForm.tsx

```diff
@@ -0,0 +1,264 @@
+              </Text>
+              <Text>{externalData?.nationalRegistry?.data?.address}</Text>
+            </Box>
+          </GridColumn>
+        </GridRow>
+      </Box>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(m.inRelationship.general.sectionTitle)}
+        editAction={() => goToScreen?.('inRelationship')}
+      >
+        {/* TODO */}
+        <Text>TODO</Text>
+      </SummaryBlock>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(
+          m.homeCircumstancesForm.general.sectionTitle,
+        )}
+        editAction={() => goToScreen?.('homeCircumstances')}
+      >
+        <Text>
+          {answers?.homeCircumstances?.type === HomeCircumstances.OTHER
+            ? answers?.homeCircumstances?.custom
+            : formatMessage(
+                getMessageHomeCircumstances[answers?.homeCircumstances?.type],
+              )}
+        </Text>
+      </SummaryBlock>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(m.studentForm.general.sectionTitle)}
+        editAction={() => goToScreen?.('student')}
+      >
+        <Text>
+          {formatMessage(getMessageApproveOptions[answers?.student?.isStudent])}
+        </Text>
+
+        {answers?.student?.isStudent === ApproveOptions.Yes && (
+          <Text marginTop={2}>{answers?.student?.custom}</Text>
+        )}
+      </SummaryBlock>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(m.employmentForm.general.sectionTitle)}
+        editAction={() => goToScreen?.('employment')}
+      >
+        <Text>
+          {answers?.employment?.type === Employment.OTHER
+            ? answers?.homeCircumstances?.custom
+            : formatMessage(
+                getMessageEmploymentStatus[answers?.employment?.type],
+              )}
+        </Text>
+      </SummaryBlock>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(m.incomeForm.general.sectionTitle)}
+        editAction={() => goToScreen?.('income')}
+      >
+        <Text>
+          {formatMessage(getMessageApproveOptionsForIncome[answers?.income])}
+        </Text>
+      </SummaryBlock>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(
+          m.summaryForm.formInfo.personalTaxCreditTitle,
+        )}
+        editAction={() => goToScreen?.('personalTaxCredit')}
+      >
+        <Text>
+          {formatMessage(getMessageApproveOptions[answers?.personalTaxCredit])}
+        </Text>
+      </SummaryBlock>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(m.bankInfoForm.general.sectionTitle)}
+        editAction={() => goToScreen?.('bankInfo')}
+      >
+        <Text>
+          {answers?.bankInfo?.bankNumber &&
+          answers?.bankInfo?.ledger &&
+          answers?.bankInfo?.accountNumber
+            ? answers?.bankInfo?.bankNumber +
+              '-' +
+              answers?.bankInfo?.ledger +
+              '-' +
+              answers?.bankInfo?.accountNumber
+            : ''}
```

Comment:
Do we want some text here if the applicant does not input the bank info?

Reply:
Already asked designer and we have agreed to have it empty, feel free to challenge it ⚔️